### DISCUSSION TITLE
Fake hesap kontrolu

### DISCUSSION THREAD

**Discussion by Enee97** [IBAX-io/go-ibax#881] [Q&A] [answered]
### Describe the bug

çok sayıda fake hesap ile airdroba katılan insanlar var. Bu konu twitter'da konusulup tavsiye ediliyor. Sisteme ip kontrolu getirilmesi gerekiyor ki daha iyi yerlere gelebilsin 

### Reproduction

Genel bir hata

### System Info

```shell
Ibax
```

### Logs

```shell
Go ibax start
```

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't [already an issue](https://github.com/IBAX-io/go-ibax/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/IBAX-io/go-ibax/discussions) or join our [Discord Chat Server](https://discord.com/invite/zRX6Mwafya).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by uludgn71** [comment]
yakın zamanda KYC bekliyorum

**Comment by komerli** [comment]
KYC gerekli

**Comment by sezginturhsn** [comment]
Kyc projenin başlangıç ve gelişmesine çok katkı sağlamayacaktır.Blok zinciri olan kabul görmüş kriptolar cüzdan ve giriş kodlarında sorumluluğu kullanıcıya bırakıyor.Projenin White paper  bu açık.Ancak cüzdan girişinde yeterli bilgilendirme olmadığını görüyorum .Test olmasından dolayı düzeltileceğini var sayıyorum.Private key almadığındam zorlandım Telegram sayfasınından destekle çözdüm.Cüzdan geliştirilmeli

**Comment by Hsnyldzc1991** [comment]
Kyc şart

**Comment by ibaxio** [comment, marked as answer]
> Kyc projenin başlangıç ve gelişmesine çok katkı sağlamayacaktır.Blok zinciri olan kabul görmüş kriptolar cüzdan ve giriş kodlarında sorumluluğu kullanıcıya bırakıyor.Projenin White paper bu açık.Ancak cüzdan girişinde yeterli bilgilendirme olmadığını görüyorum .Test olmasından dolayı düzeltileceğini var sayıyorum.Private key almadığındam zorlandım Telegram sayfasınından destekle çözdüm.Cüzdan geliştirilmeli

Thanks for your support.

IBAX is a decentralized network and the system does not record your personal information (Seed, Private Key, etc) and follows this principle of privacy, therefore we cannot make any KYC. If our users (including YOU) want to make an ECOsystem to offer some service that need people's KYC info, and you can do it as yourself.
